### ISSUE TITLE
Automatic multi-view layouts and unified play controls

### ISSUE THREAD

**Issue by Underscoreus** [darbyjohnston/DJV#250]
Heyo!

I was thinking it would be nice to maybe have the option/tools to automatically layout all the different players when not in the maximize mode. Essentially having all the different players automatically snap to an appropriate size and location on the screen depending on how many sources you want to view as well as being able to unify the play controls if you want to play all the sources simultaneously to compare them.

In other words, talking about having different video streams in rows/columns next to each other and playing in unison so it would be easy to compare two or more versions.

Thanks for all the work you do and have done so far!

**Comment by darbyjohnston** [comment]
Hi, thanks for the suggestions! I have also thought about something similar, indeed I wonder if a tiled layout should replace the existing "free-form" layout for the players entirely? I used the current layout for the players out of convenience but it definitely gets tiring moving/resizing the windows.

There's also a separate issue for synchronized playback of multiple players:
https://github.com/darbyjohnston/DJV/issues/28

**Comment by Underscoreus** [comment]
Hmm, yeah, resizing the windows every time is a little bit of an effort, especially since there seems to be a minimum size limit to the video player window. So on some machines like my 13" laptop, I am unable to actually resize two windows to fit next to each other without having them overlap, which makes it extra hard to compare two versions side by side.

I don't really mind the window feature or if you choose to make it a seamless tiled layout. My main goal was just to suggest adding in a form of automated layout since like I said I am actually not able to do that atm.

Thanks for the reply!

**Comment by darbyjohnston** [comment]
After some thought I've removed the current implementation for multiple windows; as you pointed out it wasn't working that great for comparing clips and was also causing other UI issues. I'll replace it with a more traditional A/B comparison system and unified playback controls as part of https://github.com/darbyjohnston/DJV/issues/28 and https://github.com/darbyjohnston/DJV/issues/35. I'm going to mark this issue as a duplicate and close it, but feel free to comment on those other issues if you have any more thoughts on the implementation.
